.github/dependabot.yml

```diff
@@ -45,4 +45,11 @@ updates:
           - "patch"
         exclude-patterns:
           # Add entries that we should update manually.
+          - "@datadog/libdatadog"
+          - "@datadog/native-appsec"
+          - "@datadog/native-iast-taint-tracking"
+          - "@datadog/native-metrics"
+          - "@datadog/pprof"
+          - "@datadog/sketches-js"
+          - "@datadog/wasm-js-rewriter"
```

Comment:
This will open individual PRs for these. I guess we could assign them to you in these cases or should we skip the update in general until the team opens a dedicated PR?